libexec/python/utils.py

```diff
@@ -57,13 +56,13 @@ def add_http(url,use_https=True):
     prefix = "https://"
     if use_https == False:
         prefix="http://"
-    
+
     # Does the url have http?
-    if re.search('^http*',url) == None:
+    if not url.startswith('http'):
```

Comment:
Apologies for not knowing the background. I see the discussion above, but I don't have full context.

My only point is that if we are checking to see that this is a valid URI (http or https), we should probably use the appropriate URI prefix. If that isn't needed, I would suggest taking the check out. <shrugs>